spec/requests/shows_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe ShowsController, type: :feature do
+  describe "GET shows#index" do
+    it 'renders index and has the header' do
+      visit '/'
+      expect(page).to have_text 'Smartflix'
+    end
+
+    context "when data is parsed" do
+      it 'includes the shows titles' do
+        visit "/"
+          expect(page).to have_text('My Little Pony: A New Generation')
+      end
+      it 'should limit to 20 shows' do
+        visit "/"
+        expect(page).to have_selector('h2', count: 20)
```

Comment:
there is a bit of a problem with this. Say you add some other h2 selectors in this html, this would fail. Best if you wrote the spec like:
`expect(page).to have_text('Dick Johnson Is Dead')`> this is the first title
`expect(page).to have_text('Jaguar')` > this is the 20th title
`expect(page).not_to have_text('Monsters Inside: The 24 Faces of Billy Milligan')` > this is the 21st title